### PR TITLE
Documentation for VLAN

### PR DESCRIPTION
**Pull request by ramaraosrikakulapu**
- Added initial documentation for VLAN setup - 
     Sample use case diagram
     Sample scripts
     Flags descriptions

### CODE REVIEW

File: plugins/vln/README_VLN.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Current stable versions for VLAN -
  - beta: v1beta.fukuoka.1724
```
Comment (ayasuda2OO3) [inline plugins/vln/README_VLN.md]: could you refer to the link(s) for downloading the artifact(s)?

**Review by ayasuda2OO3 — CHANGES_REQUESTED** [review]
Could you make change the file name to README.md, thus to ensure that readme file will be loaded when navigating through [the path by default](https://github.com/Enterprise-connect/ec-x-sdk/tree/vlan-doc/plugins/vln)?

**Review by palokam — CHANGES_REQUESTED** [review]
Can you be able to add the following on the top of the page?
What Problem(s)VLAN plug-in is solving?
What use cases require this plug-in?

File: plugins/vln/README_VLN.md (hunk 1 above)
Comment (ramaraosrikakulapu) [reply plugins/vln/README_VLN.md]: @ayasuda2003 .. Added the links for pointing to corresponding versions

**Comment by ayasuda2OO3** [comment]
Gayatri- Ram made you a reviewer, please help review the pending PR when you have a chance. Thank you @ramaraosrikakulapu @Gayatri212

**Comment by Gayatri212** [comment]
@ramaraosrikakulapu Documentation looks very good. This will be very helpful for setup. Just one suggestion I have can you add little information about logs as well. The vln plugin creates separate logs file so if you could provide some info about it it will be more useful for users to check if there vlan plugin is working fine.

**Review by Gayatri212 — CHANGES_REQUESTED** [review]
@ramaraosrikakulapu Documentation looks very good. This will be very helpful for setup. Just one suggestion I have can you add little information about logs as well. The vln plugin creates separate logs file so if you could provide some info about it it will be more useful for users to check if there vlan plugin is working fine.

**Review by ayasuda2OO3 — APPROVED** [review]

**Review by ayasuda2OO3 — APPROVED** [review]

**Review by ayasuda2OO3 — APPROVED** [review]

**Pull request merged**